### ISSUE TITLE
Alt Text

### ISSUE THREAD

**Issue by harvest316** [emailmonday/Cerberus#67]
I believe Alt Text should be given more attention since 66% of users read their emails with images disabled.  I believe the template should demonstrate the basics of alt text styling, and include alt text on each image.

What approach would you suggest, Ted?
